aten/src/ATen/Context.cpp

```diff
@@ -95,12 +95,28 @@ bool Context::hasLAPACK() const {
 #endif
 }
 
-at::QEngine Context::preferredQuantizedEngine() const {
+at::QEngine Context::qEngine() const {
   return quantized_engine;
 }
 
-void Context::setPreferredQuantizedEngine(at::QEngine e) {
-  quantized_engine = e;
+void Context::setQEngine(at::QEngine e) {
```

Comment:
I'd also add Context::supported_qengines which returns a list of supported ones

Reply:
is this for users to know what are the supported qengines so they can choose one?